firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/internal/util/conversions.kt

```diff
@@ -284,7 +284,7 @@ internal fun com.google.ai.client.generativeai.common.GenerateContentResponse.to
 }
 
 internal fun com.google.ai.client.generativeai.common.CountTokensResponse.toPublic() =
-  CountTokensResponse(totalTokens)
+  CountTokensResponse(totalTokens, totalBillableCharacters!!)
```

Comment:
> !!

Did you do this for testing purposes?

Reply:
No. The field is optional because genai doesn't support it, but vertex does, and, AFAIK, it's always included in that scenario. 

Thinking about it, it's probably better to check whether it's there, and throw a vertexai specific exception, rather than let the runtime die. wdyt?

Reply:
> Thinking about it, it's probably better to check whether it's there, and throw a vertexai specific exception, rather than let the runtime die. wdyt?

I was thinking the exact same thing- sgtm :)

Reply:
PTAL